### PR TITLE
Added a new troubleshooting page

### PR DESCRIPTION
**Pull request by helen-laktionova**
Added a new troubleshooting page

## PR Description

TBD

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

**Review by tvaleriy — DISMISSED** [review]

File: docs/scos/dev/troubleshooting/troubleshooting-spryker-in-cloud-issues/missing-migration-files.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+Missign migration files on production environments. 
+
+## Note
+It is recomended to push all the migrations to a repository as a go-live preparation and not during the development (to simplify the process of development). 
+
+## Solution
+
+1. Check if the files are not under ignore (f.e. .dockerignore) and added to the version control system (f.e. github).
+2. Check that during the deployment the files are not removed (f.e. check if during the deployment you are not using the ```console propel:migration:delete``` command that removes migrations).
```
Comment (feversocial) [inline docs/scos/dev/troubleshooting/troubleshooting-spryker-in-cloud-issues/missing-migration-files.md:19]: "Check that during the deployment the files are not removed"

How/where to check that? I know the answer, but not sure it is so obvious for the customer. You could mention config/install - or better reference corresp. pages. I was able to find [this one](https://docs.spryker.com/docs/cloud/dev/spryker-cloud-commerce-os/configuring-deployment-pipelines/deployment-pipelines.html) and [this one](https://docs.spryker.com/docs/cloud/dev/spryker-cloud-commerce-os/configuring-deployment-pipelines/customizing-deployment-pipelines.html)

**Review by feversocial — CHANGES_REQUESTED** [review]

File: docs/scos/dev/troubleshooting/troubleshooting-spryker-in-cloud-issues/missing-migration-files.md (hunk 1 above)
Comment (helen-laktionova) [reply docs/scos/dev/troubleshooting/troubleshooting-spryker-in-cloud-issues/missing-migration-files.md:19]: Please add it

File: docs/scos/dev/troubleshooting/troubleshooting-spryker-in-cloud-issues/missing-migration-files.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+template: ?
+originalLink: ?
+originalArticleId: ?
+redirect_from:
+---
+
+Missign migration files on production environments. 
+
+## Note
+It is recomended to push all the migrations to a repository as a go-live preparation and not during the development (to simplify the process of development).
```
[the remark this answers is not in the document]
Comment (helen-laktionova) [reply docs/scos/dev/troubleshooting/troubleshooting-spryker-in-cloud-issues/missing-migration-files.md:14]: Done - https://spryker.atlassian.net/browse/SOL-258

[the remark this answers is not in the document]
Comment (feversocial) [reply docs/scos/dev/troubleshooting/troubleshooting-spryker-in-cloud-issues/missing-migration-files.md:14]: @helen-laktionova I would consider that ticket as a blocker then. Reason: w.o. explanation why do we recommend it, existing customers that have chosen another strategy may get confused and will start asking questions.

**Review by gibmirmilch — APPROVED** [review]

**Pull request merged**